### DISCUSSION TITLE
[Issues] `Start date` and `Due date` metafields

### DISCUSSION THREAD

**Discussion by marina-chibizova** [community/community#49460] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

This issue was [requested in 2015](https://github.com/isaacs/github/issues/432) but was never released. Now that we have Projects, custom date fields partially cover needed functionality. A lot of discussions ask for Projects to automatically get these values from issue metadata and allow [ordering](https://github.com/orgs/community/discussions/8518) by dates (last updated, created, closed, due date), and community seems to be happy about this promised feature

But one crucial thing is missed here: **custom date fields like the due date are unique to each project** (and currently all date fields are unique to each project as they are manual).  If an issue is in several projects at the same time (which is the case in my organisation) date fields need to be manually filled, maintained and synchronized in all of them. It would be so much easier to have these basic dates as issue attributes, that all the projects could fetch and use.

I genuinely thunk this is super basic functionality supported by competitor software providers (Gitlab has alerts based on due date) and it should be provided on the issue level, and then project can use the date fields for ordering, roadmaps and what not.

**Comment by showhan** [comment]
+1 to this request. It's really not a good experience for users when a feature request has been submitted almost 10 years ago and still hasn't been released, whereas it's an essential feature.

I would cordially request that GitHub consider this feature, please.
